compose.yaml

```
@@ -0,0 +1,7 @@
+services:
+  db:
+    image: mysql:latest
+    ports:
+        - "3306:3306"
+    env_file:
+      - .env
```

[imo]

.envから読み込むのでも良いと思いますが、こうすると.envがすっきりするかなと思いました

```suggestion
services:
  db:
    image: mysql:latest
    ports:
        - "3306:3306"
    environment:
        MYSQL_ALLOW_EMPTY_PASSWORD: yes
        MYSQL_USER: ${DB_USER:-user}
        MYSQL_PASSWORD: ${DB_PASSWORD:-password}
        MYSQL_DATABASE: ${DB_NAME:-sushi}
```